modules/vector-index/partials/create-hyperscale-index-capella-ui.adoc

```diff
@@ -0,0 +1,85 @@
+If you choose *No*, you can still filter using a `WHERE` clause, but your index will not be partitioned based on field values.
+.. (Optional) Under *Add WHERE clause*, enter a clause for the specific values that a document must contain in your filter field to be included in your index.
++
+For example, you could filter your index so that only documents with `supplier = "HP"` or `brightness > 0.5` are included.
+. Under *Index Configuration*, choose options to configure additional settings for your HyperScale Vector index: 
+
+.. In the *Similarity Metric* list, select the similarity metric that you want to use for comparing vectors in your index.
+For the best accuracy, use the same similarity that you plan to use to query your data.
++
+For more information about vector similarity measures, see xref:vector-index:vectors-and-indexes-overview.adoc#vector_similarity[Vector Similarity].
+.. In the *Quantization* list, select the type of quantization to use for simplifying the vectors stored in your index.
++
+For more information about quantization, see xref:vector-index:vectors-and-indexes-overview.adoc#quantization[Quantization].
+For more information about how to choose quantization on your index, see xref:vector-index:vectors-and-indexes-overview.adoc#choosing_quantization[Choosing Quantization] and xref:vector-index:vector-index-best-practices.adoc#quantization[Tuning Index Creation - Quantization].
+.. In the *Dimensions* field, enter the exact dimension of the vectors in your data.
++
+Your embedding model determines this value, and Capella automatically configures it based on your chosen *Vector Field*.
+.. In the *Replicas* list, select the number of replicas you want to create for your index.
++
+Replicas affect the query throughput, memory footprint, and fault tolerance for your HyperScale Vector index.
+More replicas increase the required memory for your index, but can increase query throughput.
+.. In the *Training List* field, enter or select the number of vectors to consider when searching for centroids in your data.
++
+[TIP]
+====
+If you have less than 10,000 vectors in your index, Couchbase recommends setting the *Training List* value to your number of vectors.
+If you have more than 10,000 vectors up to a maximum of 1,000,000, use either 10% of your total number of vectors or 10 times your number of centroids - whichever value is higher, up to a maximum of 1,000,000.
+
+You can change your *Training List* value from these recommended defaults.
+Increasing the value can improve recall, but increase build times.
+Decreasing the value can improve build times, but reduces recall.
+====
+.. In the *Probes to scan* field, enter or select the number of centroids to check for similar vectors for each query.
++
+A higher value increases search times, but also increases accuracy.
+
+.. In the *Number of centroids* field, enter or select the number of centroids to create in your index.
```

Comment:
Again I think setting the number of centroids is optional. If you don't specify a value, the Index Service defaults to the number of vectors in your data divided by 1000.